File: Tshirt.java
package com.sonata.day2;



public class Tshirt {

	String color;
	String material;
	String design;
	public void display(String color, String material,String design)
	{
		this.color = color;
		this.material=material;
		this.design=design;
		
		System.out.println("Shirt color "+color+" , ");
		System.out.println("Shirt material "+material+" , ");
		System.out.println("Shirt design "+design);
		System.out.println();
	}

	public static void main(String[] args) {
		
 Tshirt small = new Tshirt();
 Tshirt large = new Tshirt();
 Tshirt xlarge = new Tshirt();
 small.display("blue", "cotton", "xyz");
 large.display("red", "silk", "abc");
 xlarge.display("black", "wool", "mno");
	}

}

File: Meghana_18310/TypeSpring/Assignment/TShirt.js
var TShirt = /** @class */ (function () {
    function TShirt(c, m, d) {
        this.color = c;
        this.material = m;
        this.design = d;
    }
    TShirt.prototype.smallSize = function () {
        return "small";
    };
    TShirt.prototype.largeSize = function () {
        return "large";
    };
    TShirt.prototype.xlSize = function () {
        return "Xtra-large";
    };
    TShirt.prototype.display = function () {
        console.log(this.color + " " + this.material + " " + this.design);
    };
    return TShirt;
}());
var shirt1 = new TShirt("blue", "cotton", "round-neck");
console.log("Size: " + shirt1.smallSize());
shirt1.display();
var shirt2 = new TShirt("red", "silk", "collar-neck");
console.log("Size: " + shirt2.largeSize());
shirt2.display();
var shirt3 = new TShirt("white", "nylon", "v-shape");
console.log("Size: " + shirt3.xlSize());
shirt3.display();
